src/DotLiquid/Strainer.cs

```diff
@@ -61,24 +78,38 @@ public Strainer(Context context)
         /// In this C# implementation, we can't use mixins. So we grab all the static
         /// methods from the specified type and use them instead.
         /// </summary>
-        /// <param name="type"></param>
-        public void Extend(Type type)
+        /// <param name="syntaxCompatibilityLevel">The Liquid syntax flag used for backward compatibility</param>
+        /// <param name="type">The type from which static methods will be extracted and mixed to the current <see cref="Strainer"/>.</param>
+        public void Extend(SyntaxCompatibility syntaxCompatibilityLevel, Type type)
         {
             // Calls to Extend replace existing filters with the same number of params.
             var methods = type.GetRuntimeMethods().Where(m => m.IsPublic && m.IsStatic);

```

Comment:
Should this be `.Where(m => m.IsPublic && !m.IsStatic && !m.IsSpecialName)` to match code in Drop?

Reply:
The line you reference wasn't changed so I'm not going to change that.